server/src/main/java/org/jboss/as/server/deployment/module/VFSResourceLoader.java

```diff
@@ -73,6 +76,18 @@
  */
 public class VFSResourceLoader extends AbstractResourceLoader implements IterableResourceLoader {
 
+    private static final String MR_PREFIX = "META-INF/releases/";
```

Comment:
This appears to be a typo. As per the javadoc of JarFile[1], the path is ```META-INF/versions/```

[1] https://docs.oracle.com/javase/9/docs/api/java/util/jar/JarFile.html

Reply:
That's what happens while you think about one part of the code while typing in another part :)  Fixed.